universal-application-tool-0.0.1/app/forms/QuestionForm.java

```diff
@@ -0,0 +1,84 @@
+package forms;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Locale;
+import java.util.Optional;
+import services.question.QuestionDefinitionBuilder;
+import services.question.QuestionType;
+
+public class QuestionForm {
+  private String questionName;
+  private String questionDescription;
+  private String questionPath;
+  private String questionType;
+  private String questionText;
+  private String questionHelpText;
+
+  public String getQuestionName() {
+    return questionName;
+  }
+
+  public void setQuestionName(String questionName) {
+    this.questionName = questionName;
+  }
+
+  public String getQuestionDescription() {
+    return questionDescription;
+  }
+
+  public void setQuestionDescription(String questionDescription) {
+    this.questionDescription = questionDescription;
+  }
+
+  public String getQuestionPath() {
+    return questionPath;
+  }
+
+  public void setQuestionPath(String questionPath) {
+    this.questionPath = questionPath;
+  }
+
+  public String getQuestionType() {
+    return questionType;
+  }
+
+  public void setQuestionType(String questionType) {
+    this.questionType = questionType;
+  }
+
+  public String getQuestionText() {
+    return questionText;
+  }
+
+  public void setQuestionText(String questionText) {
+    this.questionText = questionText;
+  }
+
+  public String getQuestionHelpText() {
+    return questionHelpText;
+  }
+
+  public void setQuestionHelpText(String questionHelpText) {
+    this.questionHelpText = questionHelpText;
+  }
+
+  public QuestionDefinitionBuilder getBuilder() {
+    ImmutableMap<Locale, String> questionTextMap =
+        (questionText.length() > 0)
```

Comment:
suggestion (blocking): this can be `!questionText.isEmpty()`

Reply:
Done.